server/routes/creatingLicences/index.ts

```diff
@@ -85,6 +88,12 @@ export default function Index({
   post('/id/:licenceId/initial-meeting-contact', initialMeetingContactHandler.POST, Telephone)
   get('/id/:licenceId/initial-meeting-time', initialMeetingTimeHandler.GET)
   post('/id/:licenceId/initial-meeting-time', initialMeetingTimeHandler.POST, SimpleDateTime)
+  get('/additional-licence-conditions-question', additionalLicenceConditionsQuestionRoutes.GET)
```

Comment:
this is just the asking whether the user needs to add additional conditions - so solely the create journey